code/src/js components/Questionamountofcoffee.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+export const Questionamountofcoffee = ({answeramount, onAmountChange}) => {
+return(
+<>
+    <div>
+        <p tabIndex='0' >How many cups of coffee do you drink per week?</p>
+            <label tabIndex='0' htmlFor="answerAmount">1-3</label>
+                <input type="radio"
```

Comment:
Hmm, your inputs are not connected to labels - I tried to click on labels and wasn't redirected.
If you add `htmlFor` attribute to label, you need to remember to add `id` with the same value to input element ;)